### PR TITLE
Add sigma to SGHMC and SGNHT

### PR DESCRIPTION
**Pull request by SamDuffield**
Adds a sigma parameter for the momenta target distribution of SGHMC and SGNHT. This is needed to get an equivalence with [SGD+momenta](https://pytorch.org/docs/stable/generated/torch.optim.SGD.html#torch.optim.SGD) when setting `temperature=0` as SGD+momenta has three tuning parameters (`lr, momentum, dampening`) assuming no weight decay.

### CODE REVIEW

**Review by KaelanDt — APPROVED** [review]
LGTM! Just curious, does the addition of sigma make the tests past more smoothly (ie lower `tol`)?
